packages/wrangler/src/__tests__/dotenv.test.ts

```diff
@@ -0,0 +1,135 @@
+import * as fs from "node:fs";
+import patchConsole from "patch-console";
+import Dev from "../dev/dev";
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { runInTempDir } from "./helpers/run-in-tmp";
+import { runWrangler } from "./helpers/run-wrangler";
+import { writeWranglerToml } from "./helpers/write-wrangler-toml";
+import type { Mock } from "vitest";
+
+runInTempDir();
+
+mockConsoleMethods();
+afterEach(() => {
+	(Dev as Mock).mockClear();
+	patchConsole(() => {});
+});
+
+describe("dotenv", () => {
+	it("should set process.env and import.meta.env when enabled in wrangler.toml", async () => {
+		writeWranglerToml({
+			main: "index.js",
+			dotenv: true,
+		});
+		fs.writeFileSync("index.js", `export default {};`);
+		fs.writeFileSync(".env", "TEST_VAR=test_value");
```

Comment:
hmm I need to verify whether we should automatically add quotes here.

Reply:
ok tested that we match dotenv behaviour correctly, they have to all be coerced into strings